amplify/backend/function/sendNotification/lib/index.ts

```diff
@@ -0,0 +1,238 @@
+// Unfortunatey we cannot destructure this library
+// https://github.com/firebase/firebase-admin-node/issues/593
+import * as admin from 'firebase-admin';
+import { Handler } from 'aws-lambda';
+
+import { graphqlRequest, notNull } from '../../utils';
+import { getParams } from '../../getParams';
+
+import {
+  Params,
+  TriggerEvent,
+  GetColonyName_SnQuery,
+  GetColonyName_SnQueryVariables,
+  GetColonyName_SnDocument,
+  GetUserDetails_SnDocument,
+  GetUserDetails_SnQueryVariables,
+  GetUserDetails_SnQuery,
+  NotificationType,
+  CreateNotification_SnMutation,
+  CreateNotification_SnMutationVariables,
+  CreateNotification_SnDocument,
+} from './types';
+
+import { notificationBuilder } from './message';
+import { sendEmail } from './email';
+
+let apiKey = 'da2-fakeApiId123456';
+let graphqlURL = 'http://localhost:20002/graphql';
+let firebaseAdminConfig;
+let mailJetApiKey;
+let mailJetApiSecret;
+
+const setEnvVariables = async () => {
+  const ENV = process.env.ENV;
+  if (ENV === 'qa') {
+    [apiKey, graphqlURL, firebaseAdminConfig, mailJetApiKey, mailJetApiSecret] =
+      await getParams([
+        'appsyncApiKey',
+        'graphqlUrl',
+        'firebaseAdminConfig',
+        'mailJetApiKey',
+        'mailJetApiSecret',
+      ]);
+  }
+};
```

Comment:
Part of this logic (api keys, configs, etc) should be fetched inside the `email` file, so it would make calling the specific messaging functions more readable.

Ie. something like:
```
await sendEmail(params);
```
Would become
```
await sendEmail(title, body);
```